tests/functions/test_origin_request.py

```diff
@@ -221,3 +221,60 @@ def test_origin_request_definitions_cache(
     mocked_time.return_value = cur_time
     obj.definitions
     assert mocked_boto3_client().query.call_count == count
+
+
+@pytest.mark.parametrize(
+    "req_uri, real_uri",
+    [
+        (
+            "/content/dist/rhel/some/filename.iso",
+            "/content/dist/rhel/some/filename.iso",
+        ),
+    ],
+    ids=[
+        "no content_type",
+    ],
+)
+@mock.patch("boto3.client")
+@mock.patch("exodus_lambda.functions.origin_request.cachetools")
+@mock.patch("exodus_lambda.functions.origin_request.datetime")
+def test_origin_request_no_content_type(
```

Comment:
Minor:
Setup for this test seems odd to me. It's using parametrize but only has one case and in that case both parameters are identical. Maybe it's meant to be expanded later? Though, we probably should only build things around what's strictly necessary rather than future potential.